graphql/typeDefs.ts

```diff
@@ -110,6 +118,18 @@ export default gql`
     updatedAt: String!
   }
 
+  type Comment {
+    id: Int!
+    fileName: String!
+    line: Int!
+    content: String!
+    authorId: Int!
+    submissionId: Int!
+    createdAt: String!
+    author: User
+    submission: Submission
```

Comment:
Shouldn't these two be not optional too?

Reply:
You're right I was thinking that it's about `schema.prisma` and not graphql `typedefs`. I made these fields non nullable as well.